ocfweb/docs/docs/about.md

```diff
@@ -32,19 +33,22 @@ resources available, the OCF also provides a variety of educational services.
 We used to co-sponsor a series of Help Sessions, short classes taught
 throughout the semester on a wide range of computing topics, from beginner to
 advanced levels, and focusing on resources available to the Berkeley campus
-community.  These now take the form of [DeCal courses][decal] offered by us. We
-also offer e-mail and in-person consulting with our volunteer staff members, as
-well as online sources of information about computing on the Berkeley campus
-and how to use the available resources.  Additionally, the OCF has conducted
-career fairs in the past with its sister organization, the [Computer Science
-Undergraduate Association][csua].
+community.  We also used to host the UNIX SysAdmin [DeCal courses][decal]. Now,
+we host regular Tech Talks that explore new technologies and how they might
+apply to elements of the OCF infrastructure. We also offer e-mail and in-person
+consulting with our volunteer staff members, as well as online sources of information
+about computing on the Berkeley campus and how to use the available resources.
+Additionally, the OCF has conducted career fairs in the past with its sister
```

Comment:
group past information about career fair and decals together, don't split it up with tech talks in between